rel/ci-config.ini

```diff
@@ -3,3 +3,8 @@
 [amqp]
 uri = "amqp://guest:guest@127.0.0.1:5672"
 pool_size = 10
+
+[log]
+syslog=debug
+console=debug
```

Comment:
What do you think about setting console to info and if an error is detected cat the syslog logs?
This way we have smaller and colored output in most cases, but a full output on failures.